src/Generator/ClassFinder.php

```diff
@@ -8,15 +8,15 @@
 use EoneoPay\Utils\DateTime;
```

Comment:
Do you think this class would be better suited to sit in utils since we use stuff like this quite often?

Reply:
Honestly Im not sure how useful this class would be in other projects - its just trying to extract all classes in a object graph?

Happy to move it if there is actually a reason for it. (In the next PR i ended up moving it into its own namespace)